test/E2ETest/Microsoft.Test.E2E.AspNet.OData/ModelBoundQuerySettings/PageAttributeTest/SkipTokenTest.cs

```diff
@@ -211,9 +215,9 @@ public async Task OrderByInExpandOnNestedProperty(string nextLink, string expect
         }
 
         [Theory]
-        [InlineData("$orderBy=Name", "Details?$orderby=Name&$skiptoken=Name-'2ndOrder',Id-2")]
-        [InlineData("$orderBy=Name desc", "$skiptoken=Name-'3rdOrder',Id-3")]
-        [InlineData("$orderBy=Name;$skip=1", "Details?$orderby=Name&$skiptoken=Name-'3rdOrder',Id-3")]
```

Comment:
Do these still work?  UrlDecoding these skip token values should be a no-op, right?